### ISSUE TITLE
Failing test with DBI 1.1.1

### ISSUE THREAD

**Issue by krlmlr** [ankane/dbx#24]
DBI 1.1.1 now quotes and binds values of class difftime correctly, this leads to a failing test in dbx. Can I release on CRAN anyway, would you like to fix dbx first?

``` r
library(dbx)
library(testthat)

db <- dbxConnect(adapter = "rsqlite", dbname = ":memory:")

dbxExecute(db, "CREATE TABLE events (id INTEGER PRIMARY KEY AUTOINCREMENT, city VARCHAR(255), counter INT, bigcounter BIGINT, speed FLOAT, distance DECIMAL, created_on DATE, updated_at DATETIME, open_time VARCHAR(255), active BOOLEAN, properties TEXT, image BLOB)")
#> [1] 0

test_that("difftime works", {
  events <- data.frame(city = as.difftime(c("12:30:55", "16:59:59")))
  dbxInsert(db, "events", events)

  res <- dbxSelect(db, "SELECT * FROM events")
  expect_equal(res$city, as.character(events$city))
})
#> ── Failure (<text>:13:3): difftime works ───────────────────────────────────────
#> res$city not equal to as.character(events$city).
#> 2/2 mismatches
#> x[1]: "12:30:55"
#> y[1]: "12.5152777777778"
#> 
#> x[2]: "16:59:59"
#> y[2]: "16.9997222222222"
```

<sup>Created on 2021-01-07 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by ankane** [comment]
Hey @krlmlr, thanks for letting me know. No need to wait on the release.

**Comment by ankane** [comment]
Addressed in 0.2.8.
